hawkular-metrics/hawkular-metrics-wrapper.sh

```diff
@@ -134,6 +104,12 @@ KEYTOOL_COMMAND=/usr/lib/jvm/java-1.8.0/jre/bin/keytool
 $KEYTOOL_COMMAND -noprompt -import -v -trustcacerts -alias kubernetes-master -file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -keystore hawkular-metrics.truststore -trustcacerts -storepass $TRUSTSTORE_PASSWORD
 popd
 
+if [ "x${JGROUPS_PASSWORD}" == "x" ]; then
+    echo "ATTENTION ATTENTION ATTENTION ATTENTION"
+    echo "The environment variable JGROUPS_PASSWORD was not set and we are generating one."
+    echo "This means that each Hawkular instance will have its own password and won't be able to form a cluster."
+    JGROUPS_PASSWORD=$(openssl rand -base64 512 | tr -dc A-Z-a-z-0-9 | head -c 17)
+fi
```

Comment:
I think we should fail here instead of continuing. If we cannot cluster using jgroups, then if we start up multiple Hawkular Metrics instances at once, they will all be updating the metrics and alerts schema at the same time which will cause a failure (one of the reasons why we cluster in the first place)